lib/std/os/windows/user32.zig

```diff
@@ -52,13 +86,21 @@ pub const WM_SYSKEYDOWN = 0x0104;
 pub const WM_SYSKEYUP = 0x0105;
 pub const WM_SYSCHAR = 0x0106;
 pub const WM_SYSDEADCHAR = 0x0107;
-pub const WM_UNICHAR = 0x0109;
```

Comment:
Why remove this?

Reply:
Copy-paste mistake, most likely. I will reinsert it.

Reply:
Due to your change of the WM_KEYLAST value, I'm curious if you copy/pasted these definitions from an old version of windows that e.g. didn't have `WM_UNICHAR` yet?

Reply:
> I'm curious if you copy/pasted these definitions from an old version of windows

Yeah, from looking into my header it's from 2010 or thereabouts. I will download the latest SDK and figure out what the current values are.